src/NServiceBus.Testing/Test.cs

```diff
@@ -20,30 +20,38 @@ public static Saga<TSaga> Saga<TSaga>() where TSaga : Saga, new()
         /// Begin the test script for the passed in saga instance.
         /// Callers need to instantiate the saga's data class as well as give it an ID.
         /// </summary>
-        public static Saga<TSaga> Saga<TSaga>(TSaga saga) where TSaga : Saga
+        public static Saga<TSaga> Saga<TSaga>(TSaga saga) where TSaga : Saga, new()
```

Comment:
we removed the new constrain here since the user already provides the saga instance. So I'm :-1: for adding it again.